app/controllers/admin/applicants_controller.rb

```diff
@@ -0,0 +1,37 @@
+class Admin::ApplicantsController < ApplicationController
+  def index
+    @applicants = Applicant.where(application_status: "Pending")
+  end
+  
+  def show
+    @applicant = Applicant.find(params[:id])
+    @applicant_pets = @applicant.pets
```

Comment:
So in the spirit of keeping our controllers lean, I would remove this and then on line 12 in your `applicants/show.html.erb` I would just call `@applicant.pets.each do`.